web/app/scripts/navbar/navbar-partial.html

```diff
@@ -0,0 +1,35 @@
+<nav class="navbar navbar-inverse" role="navigation">
+  <div class="navbar-header">
+    <a class="navbar-brand" ui-sref="home">Driver</a>
+  </div>
+  <ul class="nav navbar-nav">
+    <li class="nav-dropdown">
+      <div class="btn-group" dropdown>
+        <button id="record-type" type="button" class="btn btn-primary" dropdown-toggle>
+          {{ ctl.selectedRecordType.plural_label }} <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu" role="menu" aria-labelledby="record-type">
+          <li class="nav-rt-item" role="menuitem" ng-repeat="recordType in ctl.recordTypes | orderBy:'label'">
+            <a ng-click="ctl.selectedRecordType = recordType; ctl.updateState()">
```

Comment:
Strongly consider moving this logic to a function on the controller. It's generally frowned upon to do multiple statements in the template because it is verbose and less clear. Bonus is that the logic in the function can be tested more easily.